.github/workflows/db-test-mssql.yml

```
@@ -0,0 +1,107 @@
+on:
+  push:
+  pull_request:
+  # for now, CRON jobs only run on the default branch of the repo (i.e. usually on master)
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: "0 4 * * *"
+
+name: db-test-mssql
```

Let's go with shorter names, also for the file names.

```suggestion
name: db-mssql
```